packages/modules/manage-versions/src/components/ManageVersions/ManageVersions.test.tsx

```diff
@@ -249,7 +257,7 @@ describe("ManageVersions", () => {
     );
 
     const versionCells = container.querySelectorAll(
-      ".iui-table-body .iui-row:first-child .iui-cell"
+      ".iui-table-body .iui-table-row:first-child .iui-table-cell"
```

Comment:
same comment about targetting iui classes.. if we have here content instead, would not have any changes in future

Reply:
I might be wrong but I don't see any way of adding custom class to cells or rows